src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^[a-zA-Z0-9._%+-]+@"
+            + "[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$\n";
+
+    @Override
+    public boolean isValid(
+            String email,
+            ConstraintValidatorContext constraintValidatorContext
+    ) {
+        if (email == null) {
+            return false;
+        }
+        Pattern pattern = Pattern.compile(EMAIL_PATTERN);
```

Comment:
move this variable to constant, so you don't need to compile it every time when someone makes a request

Reply:
fixed